uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-balance-tab.component.js

```diff
@@ -152,12 +142,37 @@ define([
     }
 
     /**
+     * Updates the leave balance report using the filters passed on as parameters.
+     * The list of absence types to display is updated and the first page of the
+     * report is loaded.
+     *
+     * @param {Object} event - the component event handler.
+     * @param {Object} _filters_ - The filter values to use for updating the report.
+     * @param {Object} _filters_.absence_period - the absence period id to filter by.
```

Comment:
there are only two params: `event` and `_filters_`. The properties are just that, so it's incorrect to list them are params of the function